workflow/rules/annotate.smk

```diff
@@ -37,7 +39,9 @@ rule cyrcular_annotate_graph:
     conda:
         "../envs/cyrcular.yaml"
     params:
-        repeat_annotation=lambda wc, input: f"  --repeat-annotation {input.repeat_annotation} " if config["reference"].get("repeat_masker_download_link", "") else "",
+        repeat_annotation=lambda wc, input: f"  --repeat-annotation {input.repeat_annotation} "
+        if config["reference"].get("repeat_masker_download_link", "")
+        else "",
```

Comment:
This should probably be moved to a helper function in common.smk

Reply:
Not sure. I feel like this is a bit more readable than having to refer to `common.smk` to understand what's happening.